src/tokentree/utils/TokenTreeCheckUtils.hx

```diff
@@ -136,13 +139,13 @@ class TokenTreeCheckUtils {
 		if (token == null) {
 			return false;
 		}
-		if (token.is(DblDot)) {
+		if (token.tok.match(DblDot)) {
```

Comment:
Why not use `matches()` here?

Reply:
`matches` uses `Type.enumEq`, whereas `tok.match` uses `EnumValue` to emit an integer comparison - I assume that gives a slightly better runtime performance

I was in the middle of a refactor of tokentree, formatter and checkstyle to get rid of `Type.enumEq` comparisons since I had to touch everything anyways (to get rid of `is`), when `abstract class` dropped and required an update to tokentree. so I thought I make a quick github update just in case someone wants to use it for abstract things...

Reply:
Ah, I was mostly confused why wou use `matches()` in some cases but not others.

Reply:
yeah, `matches` is the renamed `is` function since that was deprecated in recent Haxe versions.
`matches` itself has a deprecated meta, that I disabled for PR to make it less "noisy"

the plan ist to make everything use integer comparisons, but some places match against `TokenDef`s that are not known at compile time which make it a little bit tricky.